hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -1001,7 +1001,7 @@ protected RequestFactory createRequestFactory() {
     try {
       storageClass = StorageClass.fromValue(storageClassConf);
     } catch (IllegalArgumentException e) {
-      LOG.warn("Unknown storage class property {}: {}; falling back to default storage class",
+      LOG.info("Unknown storage class property {}: {}; falling back to default storage class",
```

Comment:
If you don't configure anything and then if it is falling back to default, it should be info in that case, but if someone configured it and configured it wrong it should be warn only right? it isn't the normal flow of execution

Reply:
Ah, what a miss :(

Reply:
Thanks @ayushtkn !